hrjobcontract/js/src/job-contract/services/utils.js

```diff
@@ -87,19 +87,36 @@ define([
 
     services.factory('UtilsService', ['API','testAPI','settings','$q','$log', '$timeout', function (API, testAPI, settings, $q, $log, $timeout) {
         return {
-            getAbsenceType: function(){
+
+            /**
+             * Returns a promise that resolves to an array with including all of the Absence Types.
+             *
+             * Each returned Absence Type includes these fields:
+             * - id
+             * - title
+             * - default_entitlement
+             * - add_public_holiday_to_entitlement
+             *
+             * @returns {Promise}
+             */
+            getAbsenceTypes: function(){
                 var deffered = $q.defer();
 
-                API.resource('HRAbsenceType','get', {
-                    "return": "id,name,title"
-                }).get(function(data){
+                API.resource('AbsenceType','get', {
```

Comment:
4 space indentation instead of 2 (while you're at it, just fix the whole method). Shouldn't the .editorconfig file take over here?